src/controllers/events.controller.ts

```
@@ -101,6 +113,24 @@ export async function updateEvent(req: Request, res: Response) {
     const { id } = req.params;
     const { title, description, status } = req.body;
 
+    if (status === "PUBLISHED") {
+      const eventToValidate = await primaryClient.event.findUnique({ where: { id } });
+      if (!eventToValidate) {
+        return res.status(404).json({ success: false, error: "Event not found" });
+      }
+      const finalDescription = description !== undefined ? description : eventToValidate.description;
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)